perfkitbenchmarker/linux_benchmarks/specsfs2014_benchmark.py

```diff
@@ -0,0 +1,264 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs SPEC SFS 2014.
+
+SPEC SFS 2014 homepage: http://www.spec.org/sfs2014/
+
+In order to run this benchmark copy your 'SPECsfs2014_SP1.iso'
```

Comment:
We haven't done this for the other SPEC benchmark, but I think this should also be in the help description, or the error for data.ResourcePath should be caught and extended with SPEC-specific information about where to get and put these files.